src/components/OcDropzone.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="oc-dropzone">
+    <!-- @slot Message inside of the dropzone -->
+    <slot />
+  </div>
+</template>
+<script>
+export default {
+  name: "OcDropzone",
+  status: "review",
+  release: "0.0.1",
+}
+</script>
+
+<style lang="scss" scoped>
```

Comment:
```suggestion
<style lang="scss">
```
Could you drop the `scoped` for now, until we find out how to have scoped styles without blowing up the bundle size? :-) `system.css` size increased from 403K on master to 919K on this branch.

Reply:
Done